### PR TITLE
fix #69287 neonime.vip

### LINKED ISSUE #69287: neonime.vip
### Issue URL (Annoyance)
 [https://neonime.vip/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fneonime.vip%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.23
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads, <br/>EasyList, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy, <br/>Fanboy's Enhanced Tracking List, <br/>Fanboy Anti-Facebook List, <br/>Fanboy's Anti-thirdparty Fonts<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media, <br/>Fanboy's Social Blocking List<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese, <br/>ABPindo, <br/>Liste AR, <br/>AdBlockID, <br/>Thai Ads Filters, <br/>Frellwit's Swedish Filter

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/69287

### CODE REVIEW

File: AnnoyancesFilter/sections/widgets.txt [hunk 1]
```diff
@@ -448,7 +448,7 @@ nhentai.net##footer > p > b
 nhentai.net###chatbanner
 ! chatango.com widget
 pokemon-tube.com,shirainime.com,mangashiro.co,daemon-hentai.com,seirsanduk.us,mangashiro.org,animefrenzy.net,sedsports.com,139.99.120.222,indoxxi.show,strikeout.nu,aniwatcher.com,mylivecricket.info,debrid-file.com##iframe[src*="chatango.com"]
-||st.chatango.com^$domain=viprow.me|psntvs.me|sports-stream.net|anonenime.com|anizm.tv|hesgoal.com|sanzisport.com|goanimelist.me|pokemon-tube.com|animeindo.moe|shirainime.com|mangashiro.co|daemon-hentai.com|seirsanduk.us|mangashiro.org|animefrenzy.net|sedsports.com|139.99.120.222|indoxxi.show|cricfree.sc|cricfree.live|wespeakrugby.com|bosscast.net|strikeout.nu|aniwatcher.com|mylivecricket.info|debrid-file.com
+||st.chatango.com^$domain=neonime.vip|viprow.me|psntvs.me|sports-stream.net|anonenime.com|anizm.tv|hesgoal.com|sanzisport.com|goanimelist.me|pokemon-tube.com|animeindo.moe|shirainime.com|mangashiro.co|daemon-hentai.com|seirsanduk.us|mangashiro.org|animefrenzy.net|sedsports.com|139.99.120.222|indoxxi.show|cricfree.sc|cricfree.live|wespeakrugby.com|bosscast.net|strikeout.nu|aniwatcher.com|mylivecricket.info|debrid-file.com
```
Comment (zloyden) [inline AnnoyancesFilter/sections/widgets.txt:451]: I agree with it, but you have to put it at the end.
Have you checked this site with `AdBlockID`? It's clean for me, so we don't need rules for `specific` section.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/widgets.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/widgets.txt:451]: Confirmed, no ads with AdBlockID. I missed it's Indonesian site. Will correct soon.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
@Yuki2718 one tip for the future - if you make changes for specific sites like this one (indonesian), you should add them to the `Base filter -> foreign` section. Btw, thanks)

**Comment by Yuki2718** [comment]
Okay, got it!

**Pull request merged**